src/components/Hints/index.test.js

```diff
@@ -23,6 +23,24 @@ describe('Hints', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  test('should render jsx', () => {
+    const tree = renderer
+      .create(
+        <Hints
+          enabled
+          hints={[
+            {
+              element: '.test',
+              hint: <b>test</b>,
+            },
+          ]}
+        />
+      )
+      .toJSON();
+
+    expect(tree).toMatchSnapshot();
```

Comment:
Looks like the snapshot is `null` so it's not really useful in this case, maybe we should do something like [the equivalent test](https://github.com/HiDeoo/intro.js-react/blob/24d924f1fd69fa3cec984f4903725d0b536b3f2e/src/components/Steps/index.test.js#L29-L43) for the steps and check that at least `renderToStaticMarkup()` is even called.